docs/develop/vscode-for-scripts.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Visual Studio Code for Office Scripts (preview)
+description: How to setup the Office Scripts Code Editor to connect with VS Code on the web.
```

Comment:
nit: The official name is "VS Code for the Web" 😄 so we'd recommend saying just "VS Code" or the full "VS Code for the Web"

```suggestion
description: How to setup the Office Scripts Code Editor to connect with VS Code for the Web.
```

Reply:
Thanks @bamurtaugh. This is a slight difference from the "Excel on the web" the M365 brand guidance uses. Tagging @ElizabethSamuel-MSFT for awareness.

Reply:
We try to avoid using the 'product + platform' brand naming (e.g., VS Code for the Web) and prefer to instead refer more generically to using the product on a certain platform. If we're specifically pointing to the 'product + platform' brand (like linking to the product page), then we'd use that terminology.

Reply:
Can we please update to just "VS Code" here then? Our concern is "VS Code on the web" introduces a new term, so we'd like to consistently call it either "VS Code" or "VS Code for the Web" everywhere.

Reply:
We did. I committed your suggestion.

Reply:
Ah actually looks like the change for "VS Code for the Web" went through, so that's great. Thanks!